### ISSUE TITLE
v1.5.0 does not prompt the query results in Alfred

### ISSUE THREAD

**Issue by tsologub** [deanishe/alfred-duden#5]
In comparison to v1.4, the current version does not show the results in Alfred.
I just leave a gif here, I think it is self-explanatory. Can you please double check on your side?

![duden_workflow](https://user-images.githubusercontent.com/7289706/56850612-441cc580-6905-11e9-88c5-4195e6ddbca9.gif)

**Comment by deanishe** [comment]
> I just leave a gif here, I think it is self-explanatory

Recordings/screenshots of workflows not doing anything are pretty much useless, tbh. If you have a problem with a workflow, please post the log from Alfred's debugger.

> Can you please double check on your side?

![Image Copied on 2019-04-27 at 22 22 04 pm](https://user-images.githubusercontent.com/747913/56854692-eb681f80-693a-11e9-9ca9-24adf892c22f.png)

Try clearing the cache with `duden workflow:delcache`. It's possible you're looking at results cached by the broken, old version.

**Comment by dino115** [comment]
Hi,
i have the same issue, the debugger returns the following:

```
Starting debug for 'Duden Search'

[2019-05-06 10:30:06][ERROR: input.scriptfilter] Code 1: Traceback (most recent call last):
  File "duden.py", line 26, in <module>
    from bs4 import BeautifulSoup as BS
ImportError: No module named bs4
```

Installing the bs4 package via pip doesn't change anything. I'm not familiar with python and the python packages, but it looks like the dependencies are not installed properly.

Thanks a lot for your great workflow, cheers!

**Comment by deanishe** [comment]
@dino115 Thanks for providing the error. The workflow includes all its dependencies, but it wasn't looking in the right place.

[Should be fixed in v1.5.1](https://github.com/deanishe/alfred-duden/releases/tag/v1.5.1).
